matlab/lib/+Ice/LocalException.m

```diff
@@ -1,13 +1,21 @@
-classdef (Abstract) LocalException < Ice.Exception
+classdef LocalException < Ice.Exception
     % LocalException   Summary of LocalException
     %
-    % Base class for all Ice run-time exceptions.
-
+    % Base class for all Ice exceptions not defined in Slice.
+    %
     % Copyright (c) ZeroC, Inc. All rights reserved.
```

Comment:
I think we need an empty line between the doc comment and the copyright

https://github.com/mathworks/OpenTelemetry-Matlab/blob/1821f442af319d8333ac102e5a9d62eeda73c0b2/api/baggage/%2Bopentelemetry/%2Bbaggage/Baggage.m#L2-L4

Reply:
Fixed.